app/src/main/res/layout/fragment_profile.xml

```diff
@@ -94,11 +94,79 @@
         app:layout_constraintTop_toBottomOf="@+id/displayNameTv"
         tools:text="example@example.com" />
 
+    <com.google.android.material.card.MaterialCardView
+        android:id="@+id/premiumCv"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_margin="20dp"
+        app:cardCornerRadius="24dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/emailTv">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@drawable/rounded_cv_bordered_red"
+            android:padding="25dp">
+
+            <ImageView
+                android:id="@+id/premiumIv"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:contentDescription="@string/app_name"
+                android:src="@drawable/ic_premium"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <LinearLayout
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="20dp"
+                android:orientation="vertical"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toStartOf="@+id/navigateIv"
+                app:layout_constraintStart_toEndOf="@+id/premiumIv"
+                app:layout_constraintTop_toTopOf="parent">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Join Premium!"
+                    android:textColor="@color/red"
+                    android:textFontWeight="700"
+                    android:textSize="20sp" />
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="5dp"
+                    android:text="Enjoy watching Full-HD movies, without restrictions and without ads"
```

Comment:
Hardcoded string! should use @string resource

Reply:
Have missed them done ✅ , thansks for feedback.